src/caselawclient/xquery/is-sensitive.xqy

```diff
@@ -0,0 +1,7 @@
+xquery version "1.0-ml";
+
+declare variable $uri as xs:string? external;
+
+let $prop := fn:QName("","sensitive")
```

Comment:
the prop name here could be an external variable passed in, then you wouldn't need three almost-identical xquery files.